swan-lake/development-tutorials/test-document-the-code/test-ballerina-code/mocking.md

```diff
@@ -86,13 +81,14 @@ public function testGetRandomJoke() {
 }
 ```
 
-### Stub member functions and variables of an object
+### Stub the methods and fields of an object
 
-Instead of creating a test double, you may also choose to create a default mock object and stub the functions to return 
-a specific value or to do nothing.
+#### Stub remote methods/ normal methods
 
->**Note:** It is important to ensure that all member functions of the object being tested are properly stubbed. 
-> If any function is called within the implementation that hasn't been stubbed, the test framework will generate an 
+Instead of creating a test double, you may also choose to create a default mock object and stub the methods to return a specific value or to do nothing.
+
+>**Note:** It is important to ensure that all methods of the object being tested are properly stubbed. 
+> If any method is called within the implementation that hasn't been stubbed, the test framework will generate an 
 > error message in the following format: 
 > `no cases registered for member function '<member_function_name>' of object type '<object_type>'.`
```

Comment:
Shouldn't we use backticks around`<member_function_name>` and `<object_type>` instead of single quote?

Reply:
We have already used backticks for the complete sentence to show the sample error message.  Here `<member_function_name>` and `<object_type>` just act as placeholders. In the actual error message, we use quotes.

Reply:
```suggestion
> ``no cases registered for member function `<member_function_name>` of object type `<object_type>`.``
```
@Thevakumar-Luheerathan Since the whole sentence is within backticks, if we add single backticks around  `<member_function_name>`, it will break the syntax highlighting. Therefore we need to wrap the whole thing in duble backticks  and wrap `<member_function_name>` in single backticks as above.

Reply:
> @Thevakumar-Luheerathan Since the whole sentence is within backticks, if we add single backticks around `<member_function_name>`, it will break the syntax highlighting. Therefore we need to wrap the whole thing in duble backticks and wrap `<member_function_name>` in single backticks as above.

I think, we can't use backticks here as it represents the actual error message.  A sample error message looks like `no cases registered for member function 'get' of object type 'anon_type4'.`